tests/integration/cli/test_compile.py

```diff
@@ -186,7 +188,7 @@ def clean():
     clean()
     source_copy = temp_dir / "Interface.json"
     expected = (
-        r"ERROR: \(CompilerError\) ContractType collision\. "
+        r"ERROR *: \(CompilerError\) ContractType collision\. "
```

Comment:
```suggestion
        r"ERROR*: \(CompilerError\) ContractType collision\. "
```

Reply:
theyre saying there can be any number of spaces after ERROR